cmake/ProjectWasm3.cmake

```diff
@@ -0,0 +1,47 @@
+if(ProjectWasm3Included)
+    return()
+endif()
+set(ProjectWasm3Included TRUE)
+
+include(ExternalProject)
+
+set(prefix ${CMAKE_BINARY_DIR}/_deps)
+set(source_dir ${prefix}/src/wasm3)
+set(binary_dir ${prefix}/src/wasm3-build)
+# TODO: make sure to only copy wasm3.h
+set(include_dir ${source_dir}/source)
```

Comment:
After adding this `/source` here it fails on CI, but it still works (properly) on macos.